WooCommerce/WooCommerceTests/ViewRelated/Search/Order/OrderSearchUICommandTests.swift

```diff
@@ -0,0 +1,171 @@
+import XCTest
+import Yosemite
+@testable import WooCommerce
+import Storage
+import Networking
+import protocol WooFoundation.Analytics
+
+final class OrderSearchUICommandTests: XCTestCase {
+    let siteID: Int64 = 12345
+    private var storageManager: MockOrderStatusesStoresManager!
+    private var analyticsProvider: MockAnalyticsProvider!
+    private var analytics: WooAnalytics!
+
+    override func setUpWithError() throws {
```

Comment:
```suggestion
    override func setUp() {
```

I don't know why Apple gives the throwing variant in the template, but we usually don't need it.

Reply:
Less is more. Addressed in commit 162fb9a